src/main/java/it/gov/pagopa/pu/processecexutions/controller/IngestionFlowFileControllerImpl.java

```diff
@@ -23,9 +25,13 @@ public IngestionFlowFileControllerImpl(IngestionFlowFileRequestService service)
 
   @Override
   public ResponseEntity<Void> createIngestionFlowFile(IngestionFlowFileRequestDTO ingestionFlowFileRequestDTO) {
-    return ResponseEntity
-      .created(URI.create(String.valueOf(service.handleUploaded(ingestionFlowFileRequestDTO, SecurityUtils.getCurrentUserExternalId(), SecurityUtils.getAccessToken()).getIngestionFlowFileId())))
-      .build();
+    try {
+      return ResponseEntity
+        .created(URI.create(String.valueOf(service.handleUploaded(ingestionFlowFileRequestDTO, SecurityUtils.getCurrentUserExternalId(), SecurityUtils.getAccessToken()).getIngestionFlowFileId())))
+        .build();
+    } catch (ResourceNotFoundException e) {
+      return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
+    }
```

Comment:
don't do this, let's handle this conversion through @RestControllerAdvice component as done for all exceptions